c-bindings/BUILD.bazel

```diff
@@ -1,5 +1,7 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
 
+# gazelle:exclude ./libgojsonnet.h
```

Comment:
Consider writing a comment mentioning why this file will show up: namely, running _go build_ instead of _bazel build_. In the former case, cgo creates the file in the current directory. In the latter case, the file gets created elsewhere in Bazel's sandbox tree, so we never see it (unless you deliberately go looking around for it).